### DISCUSSION TITLE
add new lemmatization rule for english words with greek origin, normalize english and american

### DISCUSSION THREAD

**Discussion by liuxinglan** [explosion/spaCy#8571] [New Features & Project Ideas]
can we add rules for words with greek origin, like:

analysis - analyse
diagnosis - diagnose

Not sure where these rules should best be added, maybe in lemmatizer?

Thanks!!

**Comment by polm** [comment]
We don't have a good guide for this at the moment, but you can add custom rules to the lemmatizer #7102 or create an attribute ruler #7904 to do this.
